.rubocop.yml

```diff
@@ -0,0 +1 @@
+inherit_from: .rubocop_todo.yml
```

Comment:
wdyt about adding the `TargetRubyVersion` here? and maybe `DisplayCopNames` as well?

```
AllCops:
  TargetRubyVersion: 2.0
  DisplayCopNames: true
```

will probably need to add rubocop as a dev dependency too, i think version 0.50.0 is the last one to support ruby 2.0